staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/etcd.go

```diff
@@ -160,12 +160,17 @@ type StatusREST struct {
 	store *genericregistry.Store
 }
 
-var _ = rest.Updater(&StatusREST{})
+var _ = rest.Patcher(&StatusREST{})
```

Comment:
nit: keep the Updater assertion and add Patcher

Reply:
Patcher assertion implies Updater assertion https://github.com/kubernetes/kubernetes/blob/c94efcea1d5f996b0d69b31472dff581b8a25be9/staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go#L252-L256

but we don't have a lot of Updater/Patcher assertion in our code base (grep result is 1 Updater assertion and 6 Patcher assertion), so I don't know what is good convention/style

Reply:
This is fine as is, then